cmake/modules/ddr/DDRSetStub.cmake.in

```diff
@@ -145,7 +145,7 @@ function(process_source_files src_files)
 			# Adding this option to the command alleviates this issue.
 			list(APPEND pp_command "-xc++")
 		endif()
-		if("@OMR_OS_ZOS@" AND "@OMR_ENV_DATA64@")
+		if("@OMR_OS_ZOS@" AND "@OMR_ENV_DATA64@" AND NOT "@CMAKE_C_COMPILER_IS_OPENXL@")
 			# We need to set 64 bit code generation to ensure that limits.h macros are set correctly.
 			list(APPEND pp_command "-Wc,lp64")
```

Comment:
Does openxl only support 64-bit targets?

Reply:
Don't think so, but this exclusion was simply added so that the build knows not append this flag. For Open-XL, this is applied by default without specifying. Adding it to the full command with the "-Wc" format causes compilation error as it is not recognized.

Reply:
I think `CMAKE_C_COMPILER_IS_OPENXL` should not be used, in favour of using `OMR_TOOLCONFIG STREQUAL "openxl"`.